apps/src/gamelab/GameLabP5.js

```diff
@@ -348,6 +348,50 @@ GameLabP5.prototype.init = function (options) {
     this.p5decrementPreload = window.p5._getDecrementPreload.apply(this.p5, arguments);
   }.bind(this);
 
+  // Map of strings to constants for mouse states.
+  window.p5.prototype._clickKey = {
+    'leftButton': 'left',
+    'rightButton': 'right',
+    'centerButton': 'center'
+  };
+
+  // Returns a constant for a mouse state given a string.
+  window.p5.prototype._clickKeyFromString = function (buttonCode) {
+    if (this._clickKey[buttonCode]) {
+      return this._clickKey[buttonCode];
+    } else {
+      return buttonCode;
+    }
+  };
+
+  // Overrride p5.play so we can use strings in addition to constants.
+  window.p5.prototype._isMouseButtonInState = function (buttonCode, state) {
```

Comment:
I would go another step and implement this as a wrapper around the existing `_isMouseButtonInState` instead of copying the implementation over:

```javascript
// Overrride p5.play so we can use strings in addition to constants.
const p5IsMouseButtonInState = window.p5.prototype._isMouseButtonInState;
window.p5.prototype._isMouseButtonInState = function (buttonCode, state) {
  return p5IsMouseButtonInState(this._clickKeyFromString(buttonCode), state);
};
```